calico_containers/calico_ctl/node.py

```diff
@@ -319,32 +341,91 @@ def node_start(node_image, log_dir, ip, ip6, as_num, detach, kubernetes, rkt,
         environment=environment,
         host_config=host_config,
         volumes=["/proc_host",
-                 "/var/log/calico",
-                 "/usr/share/docker/plugins"])
+                 "/var/log/calico"])
     cid = container["Id"]
 
     docker_client.start(container)
-
     print "Calico node is running with id: %s" % cid
 
     if not detach:
         _attach_and_stream(container)
 
 
-def node_stop(force):
-    if force or len(client.get_endpoints(hostname=hostname,
-                                         orchestrator_id=DOCKER_ORCHESTRATOR_ID)) == 0:
-        client.remove_host(hostname)
-        try:
-            docker_client.stop("calico-node")
-        except docker.errors.APIError as err:
-            if err.response.status_code != 404:
-                raise
+def _start_libnetwork_container(etcd_authority, log_dir, libnetwork_image):
+    """
+    Start the libnetwork driver container.
 
-        print "Node stopped and all configuration removed"
-    else:
-        print "Current host has active endpoints so can't be stopped." + \
+    :param etcd_authority: The etcd authority string.
+    :param log_dir:  The log directory to use.
+    :param libnetwork_image: The name of the Calico libnetwork driver image to
+    use.  None, if not using libnetwork.
+    :return:  None
+    """
+    try:
+        docker_client.remove_container("calico-libnetwork", force=True)
+    except docker.errors.APIError as err:
+        if err.response.status_code != 404:
+            raise
+
+    environment = [
+        "HOSTNAME=%s" % hostname,
+        "ETCD_AUTHORITY=%s" % etcd_authority   # etcd host:port
+    ]
+
+    binds = {
+        log_dir:
```

Comment:
As discussed, this needs removing and a note adding to our troubleshooting/logging docs.